### PR TITLE
Wip/hughsie/fwupd install flags no history

### PR DESCRIPTION
**Pull request by hughsie**
This is needed as a prerequisite to https://github.com/hughsie/fwupd/pull/508

### CODE REVIEW

File: po/POTFILES.in [hunk 1]
```diff
@@ -8,5 +8,6 @@ plugins/uefi/fu-plugin-uefi.c
 src/fu-config.c
 src/fu-debug.c
 src/fu-main.c
+src/fu-noinst.c
```
Comment (superm1) [inline po/POTFILES.in]: This one snuck in and shouldn't be part of this PR.

**Comment by superm1** [comment]
I think you're missing the addition of the command to `fwupdmgr` in `fu-util.c` right?

File: po/POTFILES.in (hunk 1 above)
Comment (hughsie) [reply po/POTFILES.in]: Oops, thanks. Fixed.

**Comment by hughsie** [comment]
> I think you're missing the addition of the command to fwupdmgr

Too much coding today. Fixed.
